### PR TITLE
Update update-aws-observability-stack.md

### PR DESCRIPTION
**Pull request by stacykor**
2.5.1 to 2.6.0

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/deploy-use-aws-observability/update-aws-observability-stack.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Updating the Apps will create a backup of the old apps in a folder called as **
 To update an existing stack to the latest version:
 
 1. Locate the **Main Stack** created using AWS CloudFormation template and Click **Update**. <br/>  ![Stack_Step1.png](/img/observability/Stack_Step1.png)
-1. Select **Replace Current Template** and paste the URL - https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.5.1/sumologic_observability.master.template.yaml in the Amazon S3 URL option and select **Next**.
+1. Select **Replace Current Template** and paste the URL - https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml in the Amazon S3 URL option and select **Next**.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/update-aws-observability-stack.md:40]:
```suggestion
1. Select **Replace Current Template**, paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option, and then select **Next**.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor fix

**Comment by JainM6** [comment]
> Minor fix

I've fixed your suggestion.

**Review by JainM6 — APPROVED** [review]
Fixed suggestion, rest looks good.

**Pull request merged**